src/main/java/uk/gov/companieshouse/disqualifiedofficersdataapi/exceptions/InternalServerErrorException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.disqualifiedofficersdataapi.exceptions;
+
```

Comment:
Is this new class needed? The exception handler should throw an InternalServerError if it doesn't recognise the error and adding a new exception just for integration tests doesn't feel right.